### ISSUE TITLE
B7:  Scan error

### ISSUE THREAD

**Issue by UnixRonin** [Azanor/thaumcraft-beta#366]
Nether brick slabs persistently have the "Scan me, scan me!" sparkle about them, but can't be scanned.  The thaumometer just gives the "Nothing more to learn here" message.  I'm guessing this means that I've scanned them once and the sparkle is stuck "on", but I can't remember **for certain** whether I actually scanned one already or not.

**Comment by Azanor** [comment]
Very strange. I just tested and everything seemed to be working. Did you scan them as an item or placed in the world?

**Comment by UnixRonin** [comment]
Placed in the world.  The nether, actually.  A nether-brick-slab platform around the edge of my infernal furnace.  I'll try scanning one as an item and see whether that sheds light on the issue.

**Comment by UnixRonin** [comment]
OK, I broke a slab off my furnace surround and scanned it on the ground.  Still sparkly, still nothing new can be learned.  Attaching screenshot.

I *DO* have one possibly relevant mod, KleeSlabs, which allows placing vanilla slabs on end instead of flat.  However, this issue has presented itself **only** with nether brick slabs.  I will refresh my test server and retest there without KleeSlabs to see if I can duplicate it without KleeSlabs.

![2018-02-27_09 59 17](https://user-images.githubusercontent.com/5995659/36736475-9d5e21a8-1bd0-11e8-9589-0cc9d4e7b669.png)

**Comment by UnixRonin** [comment]
Oh, hey ...   that reminds me.  What is the sight-glass at the top left there supposed to be telling me?  Its meaning is not obvious.

**Comment by Zero-ui9** [comment]
It shows the amount of vis/flux in whatever chunk you're standing in, with the gold markers showing the cap for that chunk(out of I think 500). Holding shift will display numbers.

**Comment by XoviaHarmile** [comment]
Look at the tooltip, I think it's looking through the slab and instead scanning the netherrack

**Comment by jesmores** [comment]
> I think it's looking through the slab and instead scanning the netherrack

Could it be possible that KleeSlabs is somehow making it so the Thaumometer couldn't target the slab at all and so user never managed to scan it and that's why it still has the "sparkles" on it.

**Comment by Azanor** [comment]
It is possible. Kleeslabs more than likely replaces slabs completely with its own block and for some reason that block counts as transparent for raycasting.
I will look into it.

**Comment by UnixRonin** [comment]
OK, this gets weirder.

I synced everything from my live server to my test server, removed KleeSlabs from test, connected to test and scanned the nether brick slab there and found I could successfully scan it placed, but WAILA still could not "see" it if I broke it off the furnace and dropped it as an item on the ground.  (This, I'm pretty sure, is a WAILA bug, and far from the first such.  I've found at least a dozen blocks that WAILA misidentifies as different blocks.)

...Then looked at the server consoles and realized I'd mistakenly connected to my live server where KleeSlabs was still installed, **not** to the test server.

So KleeSlabs does not seem to be the actual problem here.  What's really puzzling me is why it worked today when it didn't work yesterday, without any changes to client or server.
